editoast/src/infra_cache/mod.rs

```diff
@@ -556,6 +556,17 @@ impl InfraCache {
         }
         Ok(())
     }
+
+    pub fn get_route(&self, id: &str) -> Result<&Route> {
```

Comment:
I prefer `route_id` even if it might be redundant. As you like.

Reply:
Done https://github.com/osrd-project/osrd/compare/bdac4067b9ee2bd062b7627d44b5d460cfcafab7..4a2c2f22132823ff66b3b2fdc2a3cecee6a0ad5a